.standard-GEM.md

```diff
@@ -93,7 +93,7 @@ Example: `yeast-GEM.mat`
 - [x] 🟥 `/LICENSE.md`  
 The repository must contain a license file. The default license is [CC-BY 4.0 International](https://creativecommons.org/licenses/by/4.0/). Unless a different license is desired, the file does not require modification.
 
-- [ ] 🟥 `/README.md`  
+- [x] 🟥 `/README.md`  
 The repository must contain a `README.md` file. A default file is provided, and the adequate contents must be filled in.  
 The `/README.md` file must include a version badge. A default is provided in the file.  
 Additionally, the `/README.md` file should contain the [Zenodo](https://zenodo.org) badge. As soon as the first public release is in made, the repository [should be archived via Zenodo](https://github.com/MetabolicAtlas/standard-GEM/wiki/FAQ#zenodo), and the corresponding badge be updated. A default is provided in the file.  
```

Comment:
Would you be open to enabling Zenodo? I can help to set it up.

Reply:
we can enable this later.